experimenter/tests/integration/nimbus/ios/xcodebuild.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import subprocess
+from pathlib import Path
+
+from .xcrun import XCRun
+
+here = Path(__file__).resolve()
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+
+class XCodeBuild:
+    device = os.getenv("IOS_DEVICE", "iPhone 16")
+    ios_version = os.getenv("IOS_VERSION", "18.1")
+    binary = "xcodebuild"
+    destination = f"platform=iOS Simulator,name={device},OS={ios_version}"
```

Comment:
Is there any reason to have these on the class? If theyre constants they should probably be at the top level or inside __init__ if not, since theyre being set on the class themselves.